database/ticket.go

```diff
@@ -244,6 +291,84 @@ func (vdb *VspDatabase) UpdateTicket(ticket Ticket) error {
 	})
 }
 
+// AltSigHistory retrieves the history of changes to a ticket's alternate signature.
+func (vdb *VspDatabase) AltSigHistory(ticketHash string) ([]*AltSigHistory, error) {
+	vdb.ticketsMtx.RLock()
+	defer vdb.ticketsMtx.RUnlock()
+
+	history := make([]*AltSigHistory, 0)
```

Comment:
Wouldn't this be capped to `maxAltSigs`? If that's the case then pre-allocating would be ideal, `history := make([]*AltSigHistory, 0, maxAltSigs)`

Reply:
ok